apps/widget/cypress/e2e/notifications-list.spec.ts

```diff
@@ -1,26 +1,20 @@
 describe('Notifications List', function () {
   beforeEach(function () {
-    cy.intercept('**/notifications/feed?page=0').as('getNotifications');
```

Comment:
Removing this might cause race conditions in those tests. Why do you found you need to remove it here?

Reply:
I added it now when testing why it failed, it didn't use to be there and always passed. Maybe it is worth adding if we see problems